src/main/java/org/broadinstitute/hellbender/utils/pairhmm/DragstrParams.java

```diff
@@ -0,0 +1,245 @@
+package org.broadinstitute.hellbender.utils.pairhmm;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.afcalc.AlleleFrequencyCalculator;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.afcalc.VariationalAlleleFrequencyCalculator;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class DragstrParams {
+
+    private final int maxPeriod;
+    private final int maxRepeats;
+    private final double[][] gop;
+    private final double[][] gcp;
+    private final double[][] api;
+    private final Map<Object, VariationalAlleleFrequencyCalculator> afcs;
+
+    public DragstrParams(final String path) {
+        this(openBufferedReader(path), path);
+    }
+
+    private static BufferedReader openBufferedReader(String path) {
```

Comment:
I'm torn, i think there is some sense in pulling out a `DragstrParamsTableCodec` for this file type as the majority of the methods in this class are IO methods. I think if you did that you could keep a very lightweight lookup table that is disentangled from its reading/writing given how simple this class is without having to worry about the parsing.